documentation/modules/ROOT/pages/development/engine.adoc

```diff
@@ -259,9 +259,18 @@ Allowed values are:
 
 * `Connect.class` - the output value is change event wrapping Kafka Connect's `SourceRecord`
 * `Json.class` - the output value is a pair of key and value encoded as `JSON` strings
+* `JsonByteArray.class` - the output value is a pair of key and value formatted as `JSON` and encoded as UTF-8 byte arrays
 * `Avro.class` - the output value is a pair of key and value encoded as Avro serialized records (see xref:{link-avro-serialization}[Avro Serialization] for more details)
 * `CloudEvents.class` - the output value is a pair of key and value encoded as xref:{link-cloud-events}[Cloud Events] messages
 
+The header format can also be specified when calling `DebeziumEngine#create()`.
+Allowed values are:
+
+* `Json.class` - the header values are encoded as `JSON` strings
+* `JsonByteArray.class` - the header values are formatted as `JSON` and encoded as UTF-8 byte arrays
+* `Simple.class` - the header values are encoded as strings
```

Comment:
I must admit that I am not really happy about these two options. It binds Debezium Server tightly to the way how the Connect serializes them. Maybe the plain `toString()` would suffice for most of data types?

Reply:
Just double checking that you're referring to the Simple format?  That uses the SimpleHeaderConverter from Kafka which should boil down to 
`
StringBuilder sb = new StringBuilder();
                    append(sb, value, false);
                    return sb.toString();
`

Reply:
@jeremy-l-ford Yes, that's the case. If we decide to retain that format then we should do it mostly ourselves. Let' me ask, how is for example number and string represented with Json?

Reply:
When using the JsonConverter and the schema option is not enabled, for the given inputs boolean true, int 100, and string myTest, the output is

true
100
"myTest"

Reply:
I think apicurio 2.4.1 is causing the tests to fail for some reason.  I manually set a test locally to use 2.3.1 and it passed without issue.

Reply:
It probably relates to https://github.com/Apicurio/apicurio-registry/issues/3024